documentation/asciidoc/topics/proc_backing_up_cluster.adoc

```diff
@@ -0,0 +1,53 @@
+[id='backing_up_clusters-{context}']
+= Backing Up {brandname} Clusters
+Create backup files that stores the state of {brandname} clusters to persistent volume claims.
```

Comment:
Probably stores not in PVC, but in PV. PVC is a request for PV
https://kubernetes.io/docs/concepts/storage/persistent-volumes/